v2/stacks/dataset-catalogue/Makefile

```
@@ -1,8 +1,14 @@
 # Include common make targets
 include ../common.mk
 
+# Specifies the default folder where the keys are stored in
+DEFAULT_KEYS_FOLDER ?= ../../../../dis-authentication-stub/static/keys
```

```suggestion
DEFAULT_KEYS_FOLDER ?= $(ONS_DP_SRC_ROOT)/dis-authentication-stub/static/keys
```
a few other scripts (in various repos) use `ONS_DP_SRC_ROOT` if it is set (as the root of where all your ONSdigital repos are), so that we have a common env var for this purpose

---

FYI: an alternative is something like
```suggestion
DEFAULT_KEYS_FOLDER ?= $(shell git rev-parse --show-cdup)/../dis-authentication-stub/static/keys
```

---

This is now addressed.